helm/bridge-operator-chart/templates/configmap.yaml

```
@@ -10,7 +10,7 @@ data:
         debug:
           server: true
       listen:
-        address: 'http://0.0.0.0:8000'
+        address: 'http://0.0.0.0:80000'
```

```suggestion
        address: 'http://0.0.0.0:18000'
```
`80000` > 2^16 (`65536`).